### DISCUSSION TITLE
Language per user

### DISCUSSION THREAD

**Discussion by YouHaveTrouble** [Zrips/CMI#3111] [Feature Requests]
Ability to set locale per player via commands and/or api instead of setting them for entire server.

---

Plugin as of now cannot be translated to more than 1 language at the time. This hurts multilanguage servers using CMI, since servers using it cannot be fully translated. Modern versions of minecraft allow for checking current players locale and setting plugin language 
based on that information. CMI already has multiple translation files, why not allow for all of them to be used at the same time for multiple users, making their experience better?

**Comment by thunderlight411** [comment]
sounds like a great idea, would love if this is going to be implemented
